### PR TITLE
Run coverage task when building the master branch

### PR DESCRIPTION
**Pull request by i-like-robots**
And send the data to coveralls.

https://github.com/Financial-Times/next-config-vars/pull/27

### CODE REVIEW

**Comment by matthew-andrews** [comment]
+1

also +58 EXP points in Makefile.

File: Makefile [hunk 1]
```diff
@@ -2,16 +2,20 @@ include n.Makefile
...
-	$(NPM_BIN_ENV); export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; mocha test/server/ --recursive --inline-diffs
+	$(NPM_BIN_ENV); export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; mocha 'test/server/**/*.test.js' --inline-diffs
+
++test:
+	make verify
 
-test-debug:
-	@mocha --debug-brk --reporter spec -i test/server/
+ifeq ($(CIRCLE_BRANCH),master)
+	make coverage && cat ./coverage/lcov.info | ./node_modules/.bin/coveralls
```
Comment (matthew-andrews) [inline Makefile]: can `$(NPM_BIN_ENV); coveralls` if you like.  unless you think `./node_modules/.bin` is neater then maybe we should just do that everywhere.

Comment (i-like-robots) [inline Makefile]: Not fussy ¯\_(ツ)_/¯

File: Makefile [hunk 2]
```diff
@@ -2,16 +2,20 @@ include n.Makefile
...
-
 coverage:
-	export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; istanbul cover node_modules/.bin/_mocha test/server/ -- --recursive
+	export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; istanbul cover node_modules/.bin/_mocha --report lcovonly 'test/server/**/*.test.js'
 
 unit-test:
-	$(NPM_BIN_ENV); export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; mocha test/server/ --recursive --inline-diffs
+	$(NPM_BIN_ENV); export apikey=12345; export api2key=67890; export AWS_SIGNED_FETCH_DISABLE_DNS_RESOLUTION=true; export NODE_ENV=test; mocha 'test/server/**/*.test.js' --inline-diffs
+
++test:
```
Comment (i-like-robots) [inline Makefile]: wow.

**Pull request merged**